packages/cli/src/commands/admin-web/embedded/public/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
What was the decision between having a minimum React app embedded instead of plain HTML? I agree that might've make the PR a bit too big(?).

Reply:
I believe that'll setting up React will be tackled in [this ticket](https://github.com/orgs/backstage/projects/5/views/1?pane=issue&itemId=19983621).

I wanted to keep this PR small and tight to unblock work that depended on it (for the same reason that I haven't made "dist builds" work with this yet)